axes/conf.py

```diff
@@ -118,3 +118,12 @@
 
 # set CORS allowed origins when calling authentication over ajax
 settings.AXES_ALLOWED_CORS_ORIGINS = getattr(settings, "AXES_ALLOWED_CORS_ORIGINS", "*")
+
+# set the list of sensitive parameters to cleanse from get/post data before logging
+settings.AXES_SENSITIVE_PARAMETERS = getattr(
```

Comment:
The docs mention `AXES_SENSITIVE_PARAMS` which differs from this configuration key.

Reply:
I've made the naming consistent